src/components/Navigation/TopBar/Notifications/NotificationsTemplate/NotificationTemplate.tsx

```diff
@@ -0,0 +1,230 @@
+import { FC, useMemo } from 'react';
+
+import { Button, Icon, Typography } from '@equinor/eds-core-react';
+import { delete_to_trash } from '@equinor/eds-icons';
+import { tokens } from '@equinor/eds-tokens';
+
+import { date } from '../../../../../utils';
+import OptionalTooltip from '../../../../DataDisplay/OptionalTooltip';
+import ApplicationIcon from '../../../../Icons/ApplicationIcon/ApplicationIcon';
+import { ProfileAvatar } from '../../../../index';
+import DeafultNotification from './NotificationElements/DeafultNotification';
+import ExperienceDue3Weeks from './NotificationElements/ExperienceDue3Weeks';
+import MergedBranchOrca from './NotificationElements/MergedBranchOrca';
+import ReadyToBePublished from './NotificationElements/ReadyToBePublished';
+import ReadyToReportNotification from './NotificationElements/ReadyToReportNotification';
+import RequestChangeOrca from './NotificationElements/RequestChangeOrca';
+import RequestReviewOrca from './NotificationElements/RequestReviewOrca';
+import ReviewQANotification from './NotificationElements/ReviewQANotification';
+import {
+  DefaultNotificationTypes,
+  Due3WeeksTypes,
+  ExperienceReadyToPublishTypes,
+  MergeBranchOrcaTypes,
+  NotificationsTypes,
+  ReadyToReportNotificationTypes,
+  RequestChangeOrcaTypes,
+  RequestReviewOrcaTypes,
+  ReviewQANotificationsTypes,
+} from './Notifications.types';
+
+import styled from 'styled-components';
+
+const { spacings, colors } = tokens;
+
+const Wrapper = styled.div`
+  padding: ${spacings.comfortable.medium};
+  border-bottom: 1px solid ${colors.ui.background__medium.hex};
+  &:hover {
+    background: #f2f2f2;
+  }
+`;
+
+const Content = styled.div`
```

Comment:
These styled components should also be moved to a separate file (NotificationTemplate.styles.ts)